operator-examples/keycloak.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: keycloak.org/v1alpha1
+kind: Keycloak
+metadata:
+  name: mykeycloak
+  labels:
+    app: sso
```

Comment:
Would it be a bit more clear if the label was something more like "mykeycloak"?

Reply:
Updated.